src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -13,6 +14,9 @@ export class InstructorHelpPageComponent implements OnInit {
   readonly supportEmail: string = environment.supportEmail;
   searchTerm: String = '';
   key: String = '';
+  Sections: typeof Sections = Sections;
```

Comment:
```
// enum
Sections: typeof Sections = Sections;
``` 
and shift it above this line:
`readonly supportEmail: string = environment.supportEmail;`